test/over_react_redux/connect_flux_test.dart

```diff
@@ -374,13 +374,6 @@ main() {
             'areStatePropsEqual',
           ];
 
-          const updateMethodCalls = [
-            'mapStateToProps',
-            'areStatePropsEqual',
-            'mapStateToProps',
-            'areStatePropsEqual'
-          ];
```

Comment:
Good question. I meant to ask you: why, in this case, did we previously expect double the calls?

> However, now connectFlux components and connect components (with areStatesEqual set to false) don't behave the same way.

I thought I understood what you were talking about at first but now I'm having trouble wrapping my head around it haha. Can we talk through this offline?

Reply:
Also just following up on this - if it hasn't been a priority yet and you haven't had a chance to dig in, I'd be happy to as well, just curious if you already did any discovery there :)

Reply:
Yeah I have no idea haha

Reply:
Omg I finally figured it out; it's because of the `memoizedWrapInteropValue`.

In Redux, store.state is updated, resulting in a new object every time, which busts some internal memoization caches, causing extra calls. In Flux, store.state is always the same object.

See more info here: https://github.com/Workiva/over_react/pull/462/commits/c58c3dc662240b6aa2841e85ba8d74fe891d457b